ScnSoft-Education/LeetCodeTraining/ValidParentheses19/ValidParentheses.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace LeetCodeTraining.ValidParentheses19
+{
+    public class Solution
+    {
+        public bool IsValid(string inputString)
+        {
+            Stack<char> openBrackets = new Stack<char>();
+
+            foreach (char bracket in inputString)
+            {
+                if (bracket == '(' || bracket == '{' || bracket == '[')
```

Comment:
add a special structure in which you will store your opening/closing brackets and use it in the code

Reply:
A special structure was used